### PR TITLE
install: conditionally push image to repository

### PR DESCRIPTION
**Pull request by jwtodd**
conditionally tag-and-push a built image to the specified image repository, examples being:

  minikube
  gcr.io

## PR Checklist

 - [ ] simple searchable title - `[mds-db] Add PG env var`, `[config] Fix eslint config`
 - [ ] briefly describe the changes in this PR
 - [ ] mark as draft if should not be merged
 - [ ] write tests for all new functionality

## Impacts
- [ ] Provider
- [ ] Agency
- [ ] Audit
- [ ] Policy
- [ ] Compliance
- [ ] Daily
- [ ] Native
- [ ] Policy Author

### CODE REVIEW

File: bin/install [hunk 1]
```diff
@@ -203,6 +204,13 @@ build() {
   yarn
   yarn build --ignore=@aws-lambda/*
   yarn image
+
+  if [ "${configs[image-repository]}" ]; then
+    for i in $(docker images --filter "reference=mds*latest" | grep mds | tr -s ' ' | cut -d ' ' -f 1,2 | tr ' ' ':'); do
```
Comment (oldschoolsysadmin) [inline bin/install]: Can use `docker images --format '{{ .Repository}}:{{ .Tag}}' --filter 'reference=mds-*` inside the `$()`

**Comment by jwtodd** [comment]
@invertigo @mdurling with this patch tagging-and-pushing images is now invokable via the installer cli, eg:

# note: assuming the images are already built (ie 'yarn build') one can tag-and-push the images to an image repository as follows:

% ./bin/install -c:image-repository=[repo-url] install:image

File: bin/install [hunk 2]
```diff
@@ -391,18 +406,20 @@ installIngressGatewayCertificate() {
...
+        git clone ${configs[simulator-repository]} ${configs[simulator]})
+    fi
 
-  (export PATH=/usr/local/opt/python/libexec/bin:${PATH};
-    cd ${configs[simulator]}; \
-    git pull; \
-    pip install -r requirements.txt; \
-    docker build -t trackgen .;
-    helm install ./helm --name trackgen)
+    (export PATH=/usr/local/opt/python/libexec/bin:${PATH};
```
Comment (oldschoolsysadmin) [inline bin/install]: This path is not going to exist for a lot of people

Comment (jwtodd) [reply bin/install]: likely for non-brew installs ... will take it into consideration

Comment (jwtodd) [reply bin/install]: inferring install path from $(brew --prefix python)/libexec/bin

Comment (oldschoolsysadmin) [reply bin/install]: It looks like the install path in linux is `/home/linuxbrew/.linuxbrew/bin`

Comment (jwtodd) [reply bin/install]: ok thx. i will work through this with the linux notebook in hand as well.

Comment (jwtodd) [reply bin/install]: ok $(brew --prefix python)/bin/python exists on osx and linux

**Comment by avatarneil** [comment]
Please retarget this PR at @lacuna-tech/mds-core

**Pull request closed without merging**